### PR TITLE
fix: deploy and retrieve from sfdx to sf

### PR DESCRIPTION
**Pull request by CristiCanizales**
### What does this PR do?
- Updates all code for deploy and retrieve from sfdx to sf

### What issues does this PR fix or reference?
@W-15110421@

Test run: https://github.com/forcedotcom/salesforcedx-vscode/actions/runs/8023872477 ✅

### CODE REVIEW

File: packages/salesforcedx-vscode-core/test/vscode-integration/commands/sourceDiff.test.ts
Comment (daphne-sfdc) [inline packages/salesforcedx-vscode-core/test/vscode-integration/commands/sourceDiff.test.ts:1]: Why isn't this file name changed to diff.test.ts?

Comment (CristiCanizales) [reply packages/salesforcedx-vscode-core/test/vscode-integration/commands/sourceDiff.test.ts:1]: code file is also sourceDiff :)

Comment (CristiCanizales) [reply packages/salesforcedx-vscode-core/test/vscode-integration/commands/sourceDiff.test.ts:1]: code file is also sourceDiff :)

Comment (daphne-sfdc) [reply packages/salesforcedx-vscode-core/test/vscode-integration/commands/sourceDiff.test.ts:1]: True, `diff` does sound like a short and weird name 😂

**Review by daphne-sfdc — APPROVED** [review]
Approved! 🎉 

QE notes here: https://salesforce.quip.com/SoJBAoMqtVAl

**Comment by diyer** [comment]
Great job @CristiCanizales @daphne-sfdc :)

**Pull request merged**